src/content/docs/style-guide/writing-strategies/contribute-to-site.mdx

```diff
@@ -0,0 +1,376 @@
+        <TabsBarItem id="request">
+            Make a request to the team 
+        </TabsBarItem>
+  </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="edit-browser">
+
+For most cases, we recommend editing the docs site via your browser. On every page, you can click **Edit this doc** to edit directly from your browser. 
+
+<Steps>
+    <Step>
+    ### Click **Edit this doc** 
+
+On the doc you'd like to edit, click **Edit this doc** from the right nav. This opens a text view of the doc in question, letting you view the copy alongside its code. 
+    
+  <img
+    title="Make small changes to a doc"
+    alt="Click the edit button to make small changes"
+    src={styleguideEditDoc}
+  />
+
+    </Step>
+    <Step>
+        ### Make your edits 
+
+Once you've opened the text-version of a doc, find the **pencil** icon in the top nav of the text element. This lets you interract with the doc in edit-mode rather than the default view-only mode. 
+
+    <img
+    title="Click the pencil button"
+    alt="Click the pencil button to go into edit mode"
+    src={styleguidePencilButton}
+    /> 
+
+Make your edits, and don't worry too much about formatting or grammar. We've got that covered.
+
+    </Step>
+    <Step>
+        ### Apply your changes
+
+  <img
+      title="Commit your changes"
+      alt="Click the pencil button to go into edit mode"
+      src={styleguideCommitChanges}
+  />
+
+Once you're ready to save your progress, click the **Commit changes...** button. This brings you to a page that looks like this:
+
+  <img
+    title="Write a commit message"
+    alt="Click the pencil button to go into edit mode"
+    src={styleguideCommitMessage}
+  />
+
+Let's break down what this form is asking:
+
+* **Commit message**: Required. A commit message is a brief explanation of what you changed. For example, one might read like: `Updated requirements and added content to install procedures`
+* **Extended description**: Optional, but helps the docs team review your changes faster. This space  can add additional context about the change: `Existing install procedures caused customer confusion`.
+* Always choose **Create a new branch for this commit...**. We won't accept any contribution made directly to `develop`.
+
+There are no hard and fast rules about how often you should commit. You can make as many or as few commits as you'd like.
+
+    </Step>
+    <Step>
+        ### Navigate your new pull request
+
+Your changes—that is, your  branch and its commits—will be packaged into a pull request. This essentially creates a working space for you and anyone who wants to review your work. Here's an example pull request page:
+
+  <img
+    title="Your PR and you"
+    alt="Your pull request is a working space for your contributions"
+    src={styleguidePRpage}
+  />
+
+The two main areas are **Conversation** and **Files changed**.
+
+* The **Conversation** tab is where you can see comments, like feedback from the docs team. 
```

Comment:
I think you can expand on what's in the Conversation tab (like you can see the status updates on your PR) or get a preview build on the PR. I don't think we need to explain what comments are :)